osde2e/aws_vpce_operator_tests.go

```diff
@@ -6,13 +6,52 @@ package osde2etests
 
 import (
 	"context"
+	"fmt"
+	"os"
 
 	"github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	avov1alpha2 "github.com/openshift/aws-vpce-operator/api/v1alpha2"
+	"github.com/openshift/osde2e-common/pkg/clients/openshift"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var k8s *openshift.Client
+
 var _ = ginkgo.Describe("aws-vpce-operator", func() {
-	ginkgo.It("Placeholder avo test", func(ctx context.Context) {
-		Expect(true).NotTo(Equal(false))
+	ginkgo.BeforeAll(func(ctx context.Context) {
```

Comment:
Thanks! The way the test is written currently, it'll pass on all kinds of clusters - but I am curious how to only run these tests on certain clusters nonetheless :D

Reply:
We discussed offline that the saas file for this operator has the harness test configured such that it will only run on ROSA STS clusters currently, so we won't run on GCP clusters.